### PR TITLE
Check your answers, bank holidays and service manual topic example pages

### PR DESCRIPTION
**Pull request by hannalaakso**
Adds full page examples  for testing/auditing purposes.

### CODE REVIEW

File: app/views/full-page-examples/service-manual-topic/index.njk [hunk 1]
```diff
@@ -0,0 +1,274 @@
+{# This example is based of the live "UK bank holidays" page: https://www.gov.uk/service-manual/technology #}
```
Comment (36degrees) [inline app/views/full-page-examples/service-manual-topic/index.njk]: No it's not 😛

**Comment by nickcolley** [comment]
We can also add these URLs' to the smoke tests in `app/__tests__` alongside the other examples.

**Comment by hannalaakso** [comment]
Thanks @nickcolley 🙌  

I've added tests and now using `{% set %}` for HTML variables.

File: app/views/full-page-examples/service-manual-topic/index.njk (hunk 1 above)
Comment (hannalaakso) [reply app/views/full-page-examples/service-manual-topic/index.njk]: 😄

**Pull request merged**